src/judoscale/core/config.py

```diff
@@ -1,16 +1,13 @@
 import logging
 import os
 
-logger = logging.getLogger(__name__)
-
 
 class Config:
     def __init__(self):
         self.log_level = os.environ.get("LOG_LEVEL", "INFO")
         self.dyno = os.environ.get("DYNO", None)
         self.report_interval_seconds = 10
         self.api_base_url = os.environ.get("JUDOSCALE_URL", None)
-        self._prepare_logging()
```

Comment:
Let's leave this here so the logger is set up correctly when `Config` is initialized even if `merge` is not called. We need it in both places because a caller could "merge" a different `log_level` into the config.

Reply:
I can put it back, but then we get each log in double. Why not then just leave it only in the constructor __init__?

Reply:
> Why not then just leave it only in the constructor init?

Because the caller might provide a custom `log_level` when they call `merge`.

> I can put it back, but then we get each log in double.

Oh, I didn't realize that. I think the previous implementation of `_prepare_logging` was idempotent, so we could call it multiple times without multiple side effects. Would it be difficult for the new implementation to gracefully handle multiple calls?